sidebars.js

```diff
@@ -16,6 +16,47 @@ const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
   mainSidebar: [{type: 'autogenerated', dirName: '.'}],
 
+  termtableSidebar: [
+    'index',
+    {
+      type: 'category',
+      label: 'osu! 中英翻译术语表',
+      collapsed: false,
+      items: [
+        'termtable/general',
+        'termtable/beatmap',
+        'termtable/gameplay',
+        'termtable/client',
+        'termtable/people',
+        'termtable/tech',
+        'termtable/rules',
+      ]
+    }
+  ],
+
+  lazerSidebar: [
+    'index',
+    {
+      type: 'category',
+      label: '从零开始的 osu!lazer 入门指南',
+      collapsed: false,
+      items: [
+        'lazer/lazer-tutorial/feature',
+        'lazer/lazer-tutorial/init',
+        'lazer/lazer-tutorial/setting',
+        {
+          type: 'category',
+          label: 'osu!lazer 指南',
+          collapsed: false,
+          items: [
+            'lazer/lazermap2st',
+          ],
+        },    
+      ],
+    },
```

Comment:
It seems that the auto generation feature is a bit bad, since it cannot reflect the real structure by using plain lists.

Still turning to manually type the entries ;w;